### PR TITLE
argocd: update version to v2.1.3 and include UI build

### PR DESCRIPTION
**Pull request by alexmt**
Signed-off-by: Alexander Matyushentsev <AMatyushentsev@gmail.com>

- [x] Have you followed the [guidelines for contributing](https://github.com/Homebrew/homebrew-core/blob/HEAD/CONTRIBUTING.md)?
- [x] Have you ensured that your commits follow the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit)?
- [x] Have you checked that there aren't other open [pull requests](https://github.com/Homebrew/homebrew-core/pulls) for the same formula update/change?
- [x] Have you built your formula locally with `brew install --build-from-source <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Is your test running fine `brew test <formula>`, where `<formula>` is the name of the formula you're submitting?
- [x] Does your build pass `brew audit --strict <formula>` (after doing `brew install --build-from-source <formula>`)? If this is a new formula, does it pass `brew audit --new <formula>`?

-----

PR bumps Argo CD version to v2.1.3 and add the UI build step to enable `argocd admin dashboard` command (see [docs](https://argo-cd.readthedocs.io/en/stable/user-guide/commands/argocd_admin_dashboard/))

### CODE REVIEW

File: Formula/argocd.rb [hunk 1]
```diff
@@ -15,8 +15,12 @@ class Argocd < Formula
   end
 
   depends_on "go" => :build
+  depends_on "node@14" => :build
+  depends_on "yarn" => :build
 
   def install
+    system "make", "dep-ui-local"
+    system "sh", "-c", "NODE_ENV='production' NODE_ONLINE_ENV='online' yarn --cwd ui build"
```
Comment (carlocab) [inline Formula/argocd.rb:23]:
```suggestion
    with_env(
      NODE_ENV:        "production",
      NODE_ONLINE_ENV: "online",
    ) do
      system "yarn", "--cwd", "ui", "build
    end
```

**Review by carlocab — COMMENTED** [review]
Please use the preferred commit-message style for homebrew/core. We put the name of the formula first in commit-message headings.

For new formulae:

> At Homebrew, we like to put the name of the formula up front like so: `foobar 7.3 (new formula)`.

For existing formulae:

> The preferred commit message format for simple version updates is `foobar 7.3` and for fixes is `foobar: fix flibble matrix.`.

Refer to the [commit style guide](https://docs.brew.sh/Formula-Cookbook#commit) for more details. Also, when making further changes to your pull request, adhere to the following guidelines to make sure that @BrewTestBot can merge your commits:

> - One formula per commit; one commit per formula.
> - Keep merge commits out of the pull request.

**Comment by alexmt** [comment]
Thank you for review @carlocab ! Applied your code change suggestion and updated commit message according to commit style guide. Please take another look.

**Comment by alexmt** [comment]
I've noticed that PR failed because https://argoproj.io/ is not available. I believe this is due to network issue. The https://argoproj.io/ is avaialble right now (it is hosted using github pages). It is possible to restart CI please?

**Review by carlocab — APPROVED** [review]
Thanks, @alexmt!

**Pull request closed without merging**